### ISSUE TITLE
GitHub Project Board Migration

### ISSUE THREAD

**Issue by mariaweissman** [hackforla/expunge-assist#1439]
### Overview
We need to prepare the Project Boards for the forced migration that GitHub is doing in August, so that it is already working well for the team by then.

### Action Items
- [x]  copy this template to a comment below
   ````
   ### Text from project board card
   ```
   [INSERT MARKDOWN HERE]
   ```
   - [ ] Determine where this item goes on the wiki (page and section)
   - [ ] copy item to wiki
   - [ ] hide this comment when completed.
   ````
- [x] copy markdown from a card in the "Start Here" column of the Project Board into your new comment where it says `[INSERT MARKDOWN HERE]
- [x] delete card from project board.

### Resources/Instructions
Project board URL: https://github.com/hackforla/expunge-assist/projects/1
Other team example: https://github.com/hackforla/HomeUniteUs/issues/705

**Comment by mariaweissman** [comment]
### Text from project board card
```
Team meetings
Add Type (All, Design, Dev, UX, PM, etc.), Day, Time, Duration
https://github.com/hackforla/expunge-assist/projects/1#card-69413845

```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by mariaweissman** [comment]
### Text from project board card
```
Sort the Board by Content
[All](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+ux+content+writing%22)

Sort the Board by Design
[All](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+design%22)

Sort the Board by Development
[All](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22) | [First](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22good+first+issue%22+label%3Adevelopment)

Sort the Board by Research
[All](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+research%22)

Sort the Board by Product
[All](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+product+management%22)

Sort the Board by Meeting Agendas
[All](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22feature%3A+agenda%22)
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by mariaweissman** [comment]
### Text from project board card
```

The Icebox is for issues that cannot be started because they are waiting on another issue to be completed. These issues MUST have a dependency label and link to the blocking issue.


```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by mariaweissman** [comment]
### Text from project board card
```
Prioritized Backlog

All the issue that have been approved to be worked on. Development can see only their issues by clicking on the [development label](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22)
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by mariaweissman** [comment]
### Text from project board card
```
Ready for Review
Place any issues for review here. Tag and assign reviewers.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.
